docs/pages/access-controls/sso/okta.mdx

```diff
@@ -1,107 +1,150 @@
+locate the address where Okta provides their IdP metadata to provide to Teleport.
+
+First, create a SAML 2.0 Web App in Okta:
+
+From the main navigation menu, select **Applications** -> Applications, and click
+**Create App Integration**. Select SAML 2.0, then click **Next**.
 
-Select Web Platform and SAML 2.0 Sign On Method.
 ![Create APP](../../../img/sso/okta/okta-saml-1.png)
 
-## Configure the App
+On the next screen (**General Settings**), provide a name and optional logo for
+your new app, then click **Next**. This will bring you to the **Configure SAML** section.
 
-We are going to map the Okta groups we've created above to the SAML Attribute
-statements (special signed metadata exposed via a SAML XML response).
+### Configure the App
 
-GENERAL
+Provide the following values to their respective fields:
 
-- Single sign on URL `https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
-- Audience URI (SP Entity ID)`https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
+#### General
+
+- Single sign on URL: `https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
+- Audience URI (SP Entity ID): <nobr>`https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`</nobr>
 - Name ID format `EmailAddress`
 - Application username `Okta username`
 
-SINGLE ATTRIBUTE STATEMENTS
+Replace `<cluster-url>` with your Teleport Proxy Service address or Enterprise
+Cloud tenant (e.g. `mytenant.teleport.sh`).
+
+#### Attribute Statements
 
 - Name: `username`  | Name format: `Unspecified` | Value: `user.login`
 
-GROUP ATTRIBUTE STATEMENTS
+#### Group Attribute Statements
+
+We will map our Okta groups to SAML attribute statements (special signed metadata
+exposed via a SAML XML response), so that Teleport can discover a user's group
+membership and assign matching roles.
 
 - Name: `groups` | Name format: `Unspecified`
 - Filter: `Matches regex` |  `.*`
 
-![Configure APP](../../../img/sso/okta/setup-redirection.png)
+The configuration page should now look like this:
 
-### Note: RegEx requires `.*`
+![Configure APP](../../../img/sso/okta/setup-redirection.png)
 
-![Configure APP](../../../img/sso/okta/regex.png)
+<Admonition type="warning" >
+The "Matches regex" filter requires the literal string `.*` in order to match all
+content from the group attribute statement.
+</Admonition>
 
-<Admonition
-  type="tip"
-  title="tip"
->
-  Notice that we have set "NameID" to the email format and mapped the groups with
-  a wildcard regex in the Group Attribute statements. We have also set the "Audience"
-  and SSO URL to the same value.
+<Admonition type="tip">
+Notice that we have set "NameID" to the email format and mapped the groups with
+a wildcard regex in the Group Attribute statements. We have also set the "Audience"
+and SSO URL to the same value. This is so Teleport can read and use Okta users'
+email address to create their username in Teleport, instead of relying on additional
+name fields.
```

Comment:
```suggestion
a wildcard regex in the Group Attribute statements. We have also set the "Audience"
and SSO URLs to the same value. This is so Teleport can read and use Okta users'
email addresses to create their usernames in Teleport, instead of relying on additional
name fields.
```